provisioning/roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+- include_vars: ../../common_vars.yml
+
+- name: Setup the docker repository and install docker
+  become: yes
+  shell: |
+    apt-get update
+    apt-get install apt-transport-https ca-certificates curl software-properties-common --yes
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+    apt-get update && apt-get install docker-ce --yes
+
+- name: Download docker-compose
+  become: yes
+  shell: curl -L https://github.com/docker/compose/releases/download/1.21.2/docker-compose-$(uname -s)-$(uname -m) -o /usr/local/bin/docker-compose
+
+- name: Apply executable permissions to the docker-compose binary
+  become: yes
+  shell: chmod +x /usr/local/bin/docker-compose
+
+- name: Copy docker-compose.yml
+  copy: src={{ item.src }} dest={{ item.dest }} owner=ubuntu group=ubuntu mode=0644
+  with_items:
+    - { src: '../files/docker-compose.yml', dest: '/home/ubuntu/snowplow/' }
+
+- name: Deploy snowplow mini
+  become: yes
+  shell: cd /home/ubuntu/snowplow/ && docker-compose up -d && sleep 20
+
+- name: Wait for Elasticsearch port 9200 to become open on the host, don't start checking for 10 seconds
+  wait_for:
+    port: 9200
+    delay: 10
+    sleep: 5
+    connect_timeout: 60
+
+# Kibana 5.x has issues updating index patterns
+# rebooting kibana and retrying works out
+- name: Likely to fail attempt to create ES indexes & Kibana index patterns & NSQ topics
+  become: yes
+  shell: sh {{init_dir}}/create.sh
```

Comment:
what's the reasoning for this

Reply:
Kibana 5.x had problems with creating index patterns (they still don't have a stable API documentation, unfortunately) and I guaranteed index pattern creation using a dirty way. Anyway, 6.x doesn't have aforementioned issue and I removed this part.

Reply:
nice :+1: